src/components/atoms/item/Item.css

```
@@ -0,0 +1,23 @@
+.item {
+  padding: 1rem;
+  display: flex;
+  transition: all 0.15s;

```

background를 바꾸기 위해 transition 을 추가하신거 같은데 적용되는 대상이 하나라면 이렇게 작성하면 더 보기 좋을 거 같습니다!
```suggestion
  transition: background 0.15s;

```

---

이런 방법이...! 지금까지 부트스트랩만 쓰다가 css는 처음이라 많이 헷갈리네요ㅎㅎㅎ 감사합니다!

---

all은 의도하지 않은 css animation이 동작할 수 있어서 의도한 것만 명시하는 것이 좋아요!